app/src/main/res/layout/fragment_skill_listing.xml

```diff
@@ -21,7 +21,7 @@
             android:layout_gravity="center" />
 
         <TextView
-            android:id="@+id/error_skill_fetch"
+            android:id="@+id/errorSkillFetch"
```

Comment:
Snake case should be preferred for naming IDs.

Reply:
No, it shouldn't when using those variable names in Kotlin

Reply:
@iamareebjamal 👍 Thanks for the clarification.